### PR TITLE
Add initial governance document

### PR DESCRIPTION
**Pull request by nishakm**
Signed-off-by: Nisha K <nishak@vmware.com>

### CODE REVIEW

File: GOVERNANCE.md
Comment (lachie83) [inline GOVERNANCE.md:13]: Where and how should this happen? Is there are tool that converts Gdocs to mardown that we can use to streamline this process?

File: GOVERNANCE.md
Comment (lachie83) [inline GOVERNANCE.md:17]: It might be worth link to [CODEOWNERS](https://github.com/opencontainers/wg-reference-types/blob/main/CODEOWNERS) for a list of current maintainers.

Comment (lachie83) [inline GOVERNANCE.md:17]: `The Service Level Agreement for responses is no more than 5 business days` -  Does this mean for a single maintainer to respond or all maintainers to respond?

File: GOVERNANCE.md
Comment (lachie83) [inline GOVERNANCE.md:7]: Is `neutral framework` is the best word to describe what this is? I think `documented personas` is more accurate.

File: GOVERNANCE.md
Comment (lachie83) [inline GOVERNANCE.md:1]: Can you also drop a link to this doc from the README.md?

File: GOVERNANCE.md
Comment (nishakm) [reply GOVERNANCE.md:13]: reg. where and how: Do you have any suggestions?
reg. tools: I don't know of any tools. I have previously used hackmd and etherpad to take notes. Perhaps we can migrate our note taking to one of these tools.

File: GOVERNANCE.md
Comment (nishakm) [reply GOVERNANCE.md:17]: Any maintainer to respond. I would be open to increasing the SLA time, but my view is that an SLA is a good way of holding folks accountable.

Comment (michaelb990) [inline GOVERNANCE.md:17]: I like the intent here, but I worry about an SLA being hard to enforce and not really conducive to the overall "good faith and trust" that we want to instill in maintainers and the community. I'd suggest removing the SLA for now and replace it with what contributors should do if something isn't being addressed in a timely manner. My $0.02 here would be to encourage people to bring up topics that aren't getting addressed and need attention at the weekly meeting or on Slack.

File: GOVERNANCE.md
Comment (michaelb990) [inline GOVERNANCE.md:21]: same suggestion as @lachie83 made above re: "neutral framework" => "documented personas"

**Review by jdolitsky — APPROVED** [review]

**Review by imjasonh — APPROVED** [review]

**Review by lachie83 — APPROVED** [review]

**Comment by lachie83** [comment]
Just a reminder that once we ratify this governance doc we'll have to make sure sure that notetakers PR the meeting minutes into this repository as stipulated.

File: GOVERNANCE.md
Comment (sudo-bmitch) [inline GOVERNANCE.md:13]: To do this, I'd recommend moving our note taking off of Google Docs into hackmd.io to make PR's easier (copy and paste into a markdown file in the git repo).

File: GOVERNANCE.md
Comment (sudo-bmitch) [reply GOVERNANCE.md]: I'd prefer to solve the issue of inactive maintainers on an individual basis. Otherwise we are likely to build up a list of inactive maintainers over time as people move on, and we only find out when we can't get two approvals on a PR and it's too late to have a vote to remove the inactive maintainers.

Perhaps we consider any maintainer inactive that hasn't participated in X months, with participation being a review on a PR, response on an issue, or attending a meeting. Inactive maintainers could be removed from the codeowners (but tracked in a former maintainers list), no longer counted when tallying votes, but can be reactivated if their participation resumes.

**Comment by sudo-bmitch** [comment]
Do we want to adopt a markdown style that uses one line per sentence for easier PR's? Other than that, this LGTM.

File: GOVERNANCE.md
Comment (nishakm) [reply GOVERNANCE.md:23]: Oops! Forgot this one

**Review by sudo-bmitch — APPROVED** [review]

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**
